tests/support.py

```diff
@@ -26,16 +46,20 @@ def wrapper(*args, **kw):
                     if version < min_version:
                         min_version_txt = '.'.join(map(str, min_version))
                         raise unittest.SkipTest(
-                            "Mac OS X %s or higher required, not %s"
-                            % (min_version_txt, version_txt))
```

Comment:
Please revert unrelated cosmetic changes. Unfortunately, they make the diff more noisy and make the review process harder.

Reply:
I'm on it.